scss/templates/_connection-landing.scss

```diff
@@ -11,22 +11,6 @@ $j-gutter: em(20px);
 $landing-bdr: #D9D9D9;
```

Comment:
I agree. I am, however, only removing CSS that I 100% for sure know is not used in either the old or new versions. I will make another PR as we start removing old sections.